crates/win-api-wrappers/src/process.rs

```diff
@@ -556,7 +673,18 @@ pub fn create_process_as_user(
     let application_name = application_name.map(WideString::from).unwrap_or_default();
     let current_directory = current_directory.map(WideString::from).unwrap_or_default();
 
-    let environment = environment.map(serialize_environment).transpose()?;
+    let environment = if let Some(env) = environment {
```

Comment:
Now we correctly spawn the user's environment variables for the process if environment is not explicitly specified